src/platforms/javascript/common/session-replay/troubleshooting.mdx

```diff
@@ -44,6 +44,15 @@ Another reason you might not see the body is if the format is not supported. We
 
 More details about this feature can be found in the <PlatformLink to="/session-replay/configuration/">configuration page</PlatformLink>.
 
+## Text in the iframe is not getting masked
+
+If your iframe relies on `srcdoc`, the contents are taken as-is. As opposed to loading a resource via `src`. For that reason, our masking logic doesn't run through it.
```

Comment:
```suggestion
Our masking logic doesn't run on iframe content that's provided using the `srcdoc` attribute, rather than loaded in via `src`.

To hide this content, block the iframe, as described in our Session Replay [Privacy](platforms/session-replay/privacy/#blocking) docs.
```